.eslintrc.js

```diff
@@ -1,89 +1,14 @@
-module.exports = {
-  'env': {
-    'es6': true,
-    'node': true
-  },
-  'parser': 'babel-eslint',
-  'extends': [
-    'eslint:recommended',
-    'plugin:flowtype/recommended',
-    'plugin:react/recommended',
-    'plugin:react-native/all'
```

Comment:
We might want to add these rules back (react/recommended, react-native/all). The core's eslintrc didn't have them, since the core doesn't use React, but the GUI should keep them.

Reply:
Should be added back in now

Reply:
They have been added back now with the latest commit